docs/diagrams/ParserClassUML.puml

```diff
@@ -0,0 +1,31 @@
+@startuml
+
+
+class Parser {
+-isNull (String): boolean
+-minParameterCheck(String, int): String []
++commandParser(String): String[]
++parseAddAppointment(String): Command
++parseAddDoctor(String): Command
++parseAddMedicine (String): Command
++parseAddPatient (String): Command
++parseClearExpiredMedicine(String): Command
++parseDeleteAppointment(String): Command
++parseDeleteDoctor (String): Command
++parseDeleteMedicine (String): Command
++parseDeletePatient(String): Command
++parseEditAppointment(String): Command
++parseEditDoctor (String): Command
++parseEditMedicine (String): Command
++parseEditPatient (String): Command
++parseFindAppointment(String): Command
++parseFindDoctor (String): Command
++parseFindMedicine (String): Command
++parseFindPatient (String): Command
++parseUpdateMedicineStock(String): Command
++parseViewAppointment(String): Command
++parseViewDoctor (String): Command
++parseViewMedicine(String): Command
++parseViewPatient(String): Command
+}
+@enduml
```

Comment:
Again, there seems to be too many methods displayed in this function, perhaps it will be better to reduce and display only important methods and attributes.